pkg/controller/storageoscluster/storageoscluster_controller.go

```diff
@@ -100,7 +106,9 @@ func (r *ReconcileStorageOSCluster) ResetCurrentCluster() {
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileStorageOSCluster) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	// log.Printf("Reconciling StorageOSCluster %s/%s\n", request.Namespace, request.Name)
+
+	log := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
+	log.Info("Reconciling Cluster")
```

Comment:
It is a good indicator that the reconciling is working, but I felt it was polluting the logs with something that's not very useful all the time. I think we should comment this out again if there's no good reason to keep it. I uncomment it while testing, just to be sure that reconciling is working.

```
2019-06-27T10:45:49.713Z        INFO    cluster Reconciling Cluster     {"Request.Namespace": "default", "Request.Name": "example-storageoscluster"}
2019-06-27T10:46:04.967Z        INFO    cluster Reconciling Cluster     {"Request.Namespace": "default", "Request.Name": "example-storageoscluster"}
2019-06-27T10:46:20.238Z        INFO    cluster Reconciling Cluster     {"Request.Namespace": "default", "Request.Name": "example-storageoscluster"}
2019-06-27T10:46:35.618Z        INFO    cluster Reconciling Cluster     {"Request.Namespace": "default", "Request.Name": "example-storageoscluster"}
201
```